src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -285,6 +288,9 @@ function ReportActionItemSingle({
                         <ReportActionItemDate created={action?.created ?? ''} />
                     </View>
                 ) : null}
+                {delegateEmail && !isReportPreviewAction && (
```

Comment:
should we use action?.delegateAccountID instead of delegateEmail

Reply:
tbh, I don't see any reason why we need to use delegateEmail. There is no difference between using action?.delegateAccountID and delegateEmail. I suggest using delegateAccountID for more cleaner

Reply:
delegateAccountID is only used for logs and stuff on the backend.

we should use `delegateEmail` by default

Reply:
@rushatgabhane In this case, delegateEmail is retrieved from delegateAccountID

```
    const delegatePersonalDetails = personalDetails[action?.delegateAccountID ?? ''];

    const delegateEmail = delegatePersonalDetails?.login;
```

Reply:
oh lol mb!

Reply:
@allgandalf Any thought on this comment?

Reply:
@allgandalf One left comment here, everything else looks good